pyquil/tests/test_api.py

```diff
@@ -249,65 +242,24 @@ def test_get_qc_returns_remote_qvm_compiler():
         assert isinstance(qc.compiler, QVMCompiler)
 
 
-mock_compiler_server = Server()
-
-
-@mock_compiler_server.rpc_handler
-def get_version_info() -> dict:
-    time.sleep(0.1)
-    return {'compiler_server': '1.0.0'}
-
-
-@mock_compiler_server.rpc_handler
-def quil_to_native_quil(payload: NativeQuilRequest) -> NativeQuilResponse:
-    assert payload.quil == BELL_STATE.out()
-    time.sleep(0.1)
-    return NativeQuilResponse(quil=COMPILED_BELL_STATE.out(),
-                              metadata=NativeQuilMetadata(final_rewiring=[],
-                                                          gate_depth=0,
-                                                          gate_volume=0,
-                                                          multiqubit_gate_depth=0,
-                                                          program_duration=0.0,
-                                                          program_fidelity=0.0,
-                                                          topological_swaps=0))
+mock_qpu_compiler_server = Server()
 
 
-@mock_compiler_server.rpc_handler
+@mock_qpu_compiler_server.rpc_handler
 def native_quil_to_binary(payload: BinaryExecutableRequest) -> BinaryExecutableResponse:
     assert payload.quil == COMPILED_BELL_STATE.out()
     time.sleep(0.1)
     return BinaryExecutableResponse(program=COMPILED_BYTES_ARRAY)
 
 
-@mock_compiler_server.rpc_handler
```

Comment:
can we reuse any of these to test against quilc?

Reply:
It looks like these are tested against a live local quilc if one is available, and he's only dropping the mock versions. Doesn't seem too unreasonable.